SeaPkg/Include/Library/PeCoffLibNegative.h

```diff
@@ -36,6 +36,8 @@
 #define IMAGE_VALIDATION_DATA_SIGNATURE   SIGNATURE_32 ('V', 'A', 'L', 'D')
 #define IMAGE_VALIDATION_ENTRY_SIGNATURE  SIGNATURE_32 ('E', 'N', 'T', 'R')
 
+#define MAX_ENTRY_SIZE 80

```

Comment:
Why is this 80? Is it in the unit of bytes?

Reply:
Closing this comment as we decided the change was messy.